src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx

```diff
@@ -0,0 +1,363 @@
+---
+
+Use the New Relic mobile SDK API calls to customize and extend the data your mobile app reports to New Relic. 
+
+While our mobile monitoring solution offers powerful out-of-the-box data collection, you may need to customize and extend the data your mobile app reports to New Relic. For example, you can use the public SDK API calls to:
+
+    * Start and stop interaction traces from events in your mobile app
+    * Record custom metrics
+    * Send custom attributes and events
+    * Track network requests from libraries not supported automatically
+    * Set custom IDs to user sessions
+
+Most of these customizations are available for both native (iOS and Android) and hybrid (React Native, Xamarin, etc.) mobile apps.
+
+## Get started [#get-started]
+
+1. Make sure you have the latest mobile agent installed.
+2. Review our guidelines for using iOS and Android SDK methods:
+
+    * [Android SDK usage](#ios-sdk-usage)
+    * [iOS SDK usage](#android-sdk-usage)
+
+3. Select a customization from the table below and follow the method's documentation.
+
+## Available customizations [#methods]
+
+The table below lists some supported SDK methods to customize your native or hybrid agent instrumentation.
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "300px" }}>
+        If you want to...
+      </th>
+
+      <th>
+        Use this method
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td id="crash-analysis">
+        Track app activity that may be helpful for troubleshooting crashes.
+      </td>
+
+      <td>
+        [Record a breadcrumb](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb)
+      </td>
+    </tr>
+
+    <tr>
+      <td id="creating">
+        Track a method as an interaction.
+      </td>
+
+      <td>
+        [Start interactions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/start-interaction)
+
+        [Stop interactions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/stop-interaction)
+      </td>
+    </tr>
+
+    <tr>
+      <td id="naming">
+        Name or rename a custom interaction.
+      </td>
+
+      <td>
+        [Set the interaction name](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/name-interactions).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Set an app version associated with an interaction.
+      </td>
+
+      <td>
+        [Set the application version](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-application-version/)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Set your app's build ID.
+      </td>
+
+      <td>
+        [Set the build ID](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-application-build-number/)
+      </td>
+    </tr>
+
+    <tr>
+      <td id="create-custom">
+        Create custom metrics.
+      </td>
+
+      <td>
+        [Record custom metrics](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-metrics/).
```

Comment:
The link looks fine, but for some reason when you click, it directs to: "/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-metrics/"

Reply:
Ah yes, it's missing the backslash before docs/mobile-monitoring. Let's see if that will fix it